Kernel/FileSystem/FileSystem.h

```diff
@@ -60,6 +61,8 @@ class FS : public Retainable<FS> {
 
     virtual void flush_writes() {}
 
+    void lock_permanently() { m_lock.lock(); }
```

Comment:
Little sad about exposing this, but I'm not sure how to better do it, since the lock mangling is typically internal to the FS, but now we need to lock everything from the outside?

Reply:
Well you're still locking from an `FS` member function, so can't you just access `m_lock` directly?

Reply:
Hmm... I'll try!

Reply:
Yep, of course! :)